sw/device/lib/arch/device_sim_dv.c

```diff
@@ -0,0 +1,18 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/arch/device.h"
+
+/**
+ * Device-specific symbol definitions for the Verilator device.
+ */
+
+const device_type_t kDeviceType = kDeviceSimDV;
+
+// TODO: DV testbench completely randomizes these. Need to add code to
+// retrieve these from a preloaded memory location set by the testbench.
+
+const uint64_t kClockFreqHz = 50 * 1000 * 1000;  // 50MHz
+
+const uint64_t kUartBaudrate = 2097152; // 2Mbps
```

Comment:
Is this megabit or mebibit? It'd be useful if this was re-written as `2 * (1 << 20)` or something.

Reply:
its Mib/s indeed. updated.